octoprint_mrbeam/analytics/usage_handler.py

```diff
@@ -356,36 +358,59 @@ def _load_usage_data(self):
                 "Trying to recover from _backup_file file: %s", self._backup_file
             )
             recovery_try = True
-            if os.path.isfile(self._backup_file):
+            try:
+                with open(self._backup_file, "r") as stream:
+                    data = yaml.safe_load(stream)
+                if self._validate_data(data):
+                    data["restored"] = (
+                        data["restored"] + 1 if "restored" in data else 1
+                    )
+                    self._usage_data = data
+                    success = True
```

Comment:
is it safe to set this variable to True before writing to the file ?

Reply:
I can change the order